### ISSUE TITLE
Train only Image Reconstruction Model

### ISSUE THREAD

**Issue by ayushmankumar7** [sconlyshootery/FeatDepth#81]
Hello, Thank you for such a wonderful paper. 
I was working on Image reconstruction. Really liked your regularization using smooth_loss1 and smooth_loss2. 

So after running a single epoch, we get a loss_dict from the Autoencoder which has the smooth loss and reconstruction loss(L1 + SSIM) at 4 different scales. My question is how is backpropagation working while we train the Image Reconstruction network?

Thank you!

**Comment by sconlyshootery** [comment]
The total loss is the sum of all the losses.

**Comment by ayushmankumar7** [comment]
Thanks for replying! 
Yes, I saw that in your code.
